gnomad_qc/v2/resources/sample_qc.py

```diff
@@ -1,17 +1,61 @@
-from .basics import DataException
+from .basics import DataException, get_gnomad_meta
+from gnomad.utils.liftover import get_liftover_genome
+import hail as hl
 
 
-def qc_mt_path(data_type: str, ld_pruned: bool = False) -> str:
+def get_liftover_v2_qc_mt(
+        data_type: str,
+        ld_pruned: bool,
+        release_only: bool = False,
+        overwrite: bool = False
+) -> hl.MatrixTable:
     """
-    Returns MatrixTable for sample QC purposes: can be exomes, genomes, or joint (joint dataset can also be ld_pruned=True)
+    Returns MatrixTable for sample QC purposes on build 38: can be exomes, genomes, or joint (joint dataset can also be ld_pruned=True)
+    Criteria: callrate > 0.99, AF > 0.001, SNPs only, bi-allelics only
+    Note: sites where the locus changes chromosome are discarded
+    """
+    path = qc_mt_path(data_type, ld_pruned, 'GRCh38')
+    if not overwrite and hl.hadoop_exists(path):
+        grch38_qc_mt = hl.read_matrix_table(path)
+    else:
+        grch38_qc_mt = hl.read_matrix_table(qc_mt_path(data_type, ld_pruned=ld_pruned))
+        get_liftover_genome(grch38_qc_mt)
+        grch38_qc_mt = grch38_qc_mt.key_rows_by()
+        grch38_qc_mt = grch38_qc_mt.transmute_rows(
+            locus=hl.liftover(grch38_qc_mt.locus, 'GRCh38'),
+            locus37=grch38_qc_mt.locus
+        )
+        grch38_qc_mt = grch38_qc_mt.filter_rows(
+            grch38_qc_mt.locus.contig == 'chr' + grch38_qc_mt.locus37.contig
+        ).key_by('locus', 'alleles')
+        grch38_qc_mt = grch38_qc_mt.key_rows_by(locus=grch38_qc_mt.locus38, alleles=grch38_qc_mt.alleles)
+        grch38_qc_mt = grch38_qc_mt.checkpoint(path, overwrite=overwrite)
+
+    if release_only:
+        meta = get_gnomad_meta(data_type)
+        grch38_qc_mt = grch38_qc_mt.filter_cols(meta[grch38_qc_mt.col_key].release)
+        grch38_qc_mt = grch38_qc_mt.select_cols()
```

Comment:
why do you need this `select_cols()`? are there column fields you need to drop?

Reply:
no, you're right -- this was a merging oversight!